packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { IButtonStyles } from '../Button.Props';
+import {
+  ITheme,
+  getTheme,
+  mergeStyles,
+  mergeStyleSets
+} from '../../../Styling';
+import {
+  getStyles as getDefaultButtonStyles
+} from '../DefaultButton/DefaultButton.styles';
+
+export function getStyles(
```

Comment:
not saying to change any of the functionality. custom themes are obviously important. Just curious if we can wrap it up and abstract what I'm sure will end up in just about every styles file. 

If you think this is about as abstract as you want to get right now, no worries. We can always tweak it later.